internal/kldevents/subscription.go

```diff
@@ -43,13 +43,14 @@ type ethFilter struct {
 // SubscriptionInfo is the persisted data for the subscription
 type SubscriptionInfo struct {
 	kldmessages.TimeSorted
-	ID        string                     `json:"id,omitempty"`
-	Path      string                     `json:"path"`
-	Name      string                     `json:"name"`
-	Stream    string                     `json:"stream"`
-	Filter    persistedFilter            `json:"filter"`
-	Event     kldbind.MarshalledABIEvent `json:"event"`
-	FromBlock string                     `json:"fromBlock,omitempty"`
+	ID         string                     `json:"id,omitempty"`
+	Path       string                     `json:"path"`
+	Name       string                     `json:"name"`
+	CustomName string                     `json:"customName,omitempty"`
```

Comment:
I think it's going to be confusing to have `customName` here.
I'm guessing the problem was we auto-generate the `name` field previously from combination of the event signature and the scope - which is useful information regardless of whether a user has a custom name.

With that in mind, please can we make the _new_ field `summary`, and have that always be auto-generated (cannot be user-submitted - works just like `name` does today). Then `name` defaults to the value of `summary` if it's empty/null, but if supplied the user can put anything they like?

Reply:
Added a new field `summary` (not returned in API response). `name` is now set to whatever is provided by the user or is set to `summary` if it is empty.